### DISCUSSION TITLE
Implementing Peer Server on NestJS no longer works

### DISCUSSION THREAD

**Discussion by deviate-dv8** [peers/peerjs#1227] [Q&A] [answered]
Hello everyone, I followed the docs for setting up peerjs-server on NestJS from https://docs.peerjs.com/server/frameworks/nestjs .But unfortunately It no longer works (or maybe for me). I started a fresh nestjs application, with only peer-server.service.ts being created. and i have put it as providers in AppModule. but when i localhost:3000/peer-server it shows 404 even in clients and postman. Did I do something wrong?

**Comment by MicroMatrixOrg** [comment]
> Hello everyone, I followed the docs for setting up peerjs-server on NestJS from https://docs.peerjs.com/server/frameworks/nestjs .But unfortunately It no longer works (or maybe for me). I started a fresh nestjs application, with only peer-server.service.ts being created. and i have put it as providers in AppModule. but when i localhost:3000/peer-server it shows 404 even in clients and postman. Did I do something wrong?

I had the problem. did you solve it ?

**Comment by MicroMatrixOrg** [comment, marked as answer]
> Hello everyone, I followed the docs for setting up peerjs-server on NestJS from https://docs.peerjs.com/server/frameworks/nestjs .But unfortunately It no longer works (or maybe for me). I started a fresh nestjs application, with only peer-server.service.ts being created. and i have put it as providers in AppModule. but when i localhost:3000/peer-server it shows 404 even in clients and postman. Did I do something wrong?

I had soled the proplem.
Add the code to your main.ts.
![image](https://github.com/peers/peerjs-server/assets/21361701/7069304d-577e-4084-8155-7c04bd9cec9a)

**Reply by deviate-dv8** [reply]
Hello, Thank you for providing the solution. I also have the same answer but I forgot to send it here
